gio/src/list_store.rs

```diff
@@ -65,6 +65,48 @@ impl ListStore {
     pub fn extend_from_slice(&self, additions: &[impl IsA<glib::Object>]) {
         self.splice(self.n_items() - 1, 0, additions)
     }
+
+    #[cfg(any(feature = "v2_74", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_74")))]
+    #[doc(alias = "g_list_store_find_with_equal_func_full")]
+    #[doc(alias = "g_list_store_find_with_equal_func")]
+    pub fn find_with_equal_func<F: FnMut(&glib::Object, &glib::Object) -> bool>(
```

Comment:
This one is a bit strange. From a Rust point of view we would probably like to have a closure `FnMut(&glib::Object) -> bool` as the second object is literally just `item` that is passed through every time.

In Rust we have proper closures, such hacks are not really needed.

Reply:
Changed it accordingly, but opinions here? @bilelmoussaoui @jf2048 ?

Reply:
Seems fine to me, definitely more in line with `std::iter::Iterator::find`